ansible/roles/vm_set/tasks/bind_vm.yml

```diff
@@ -0,0 +1,38 @@
+- name: Bind VM {{ vm_name }}
+  vm_topology:
+    cmd: "bind"
+    vm_set_name: "{{ vm_set_name }}"
+    topo: "{{ topology }}"
+    vm_names: "{{ VM_targets }}"
+    vm_name: "{{ vm_name }}"
```

Comment:
In vm_topology.py, the `vm_name` argument is defined as a list. Here, it looks like a string is passed in as value for the `vm_name` argument. This is confusing too.